lib/ansible/modules/rpm_key.py

```diff
@@ -20,7 +20,7 @@
 options:
     key:
       description:
-        - Key that will be modified. Can be a url, a file on the managed node, or a keyid if the key
+        - Key that will be modified. Can be an HTTP, HTTPS or FTP URL, a file on the managed node, or a keyid if the key
```

Comment:
no ftps?

Reply:
urllib don't allow ftps, since urlibs is used to fetch and open urls, ftps will not be compatible